pkg/releaser/git.go

```diff
@@ -129,6 +129,12 @@ func (r *Releaser) submitPR(request *source.ReleaseRequest) (string, error) {
 	tc := oauth2.NewClient(context.TODO(), ts)
 	client := github.NewClient(tc)
 
+	// Close outdated PRs before submiting new one
```

Comment:
can you please pull this out of submitPR function. We can create github client outside of the function.

Reply:
OK, moved to utils.go just before the submitPR call